app/services/weet.js

```diff
@@ -0,0 +1,23 @@
+const { default: Axios } = require('axios');
+const logger = require('../logger');
+
+/**
+ * @description call the geek-jokes api
+ * @typedef {Object} GeekJoke
+ * @property {string} joke
+ * @returns {GeekJoke} return a joke
+ */
+async function getWeet() {
+  try {
+    const apiUrl = 'https://geek-jokes.sameerkumar.website/api?format=json';
+    const response = await Axios.get(apiUrl);
```

Comment:
you can simplify a bit by returning directly `return (await Axios.get (apiUrl)). data;`